### DISCUSSION TITLE
More than one heading in one chunk output not rendered correctly

### DISCUSSION THREAD

**Discussion by etschste** [quarto-dev/quarto-cli#8407] [Q&A] [answered]
### Description

I tried to create sections from a chunk. It seems that I can't cat() two sections inside one statement. It's working when there are seperate statements. 

````qmd

---
format: html
---

# Working, headings will be created

```{r}
#| results: asis
#| echo: false
cat("## Heading\n\n")
cat("## Another Heading\n\n")
```

# Not working, second heading is not rendered

```{r}
#| results: asis
#| echo: false
cat("## Heading\n\n",
    "## Another Heading\n\n")
```
````

Regards, Stefan

**Comment by mcanouil** [comment, marked as answer]
Thanks for the reproducible example but it has actually nothing to do with Quarto.
`cat()` has `sep = " "` by default, so what you are writing is:

```md
## Heading

 ## Another Heading
```

This is not a valid header.

Use `keep-md: true` to look at the markdown you generate.
